rpcs3/Emu/RSX/VK/VKProgramHelper.hpp

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include "VKProgramPipeline.h"
+
+namespace vk
+{
+	namespace glsl

```

Comment:
What's this file?

Reply:
A mistake. This should not be in the PR - oops